### PR TITLE
[ABANDONED] add xing buttons

### LINKED ISSUE #274: xing share-button
Xing share-button op de website
https://www.xing.com/companies/defactosoftware

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Fixes #274

- [x] xing share icon 
- [ ] xing company profile icon

### CODE REVIEW

File: source/localizable/_social-share.erb [hunk 1]
```diff
@@ -45,5 +45,17 @@
         <span><%= t("blog.share.linkedin") %></span>
       </a>
     </li>
+    <% if I18n.locale == :de %>
```
Comment (fatboypunk) [inline source/localizable/_social-share.erb]: do we explicitly not want to have this on the dutch page?
Also an indent after the if might be nice :)

**Comment by fatboypunk** [comment]
can't find the icon :/ Where else should i take a look?
screen shot 2016-07-25 at 22 58 45

**Comment by RonaldDijkstra** [comment]
Beneath blog.

**Comment by fatboypunk** [comment]
Only place I didn't check, do we only want it there?

**Comment by RonaldDijkstra** [comment]
No, it's gonna be in the footer too.

**Comment by fatboypunk** [comment]
But that's for annother PR?

**Comment by RonaldDijkstra** [comment]
Yes.

**Comment by RonaldDijkstra** [comment]
I thought it would be nice to add a Google+ icon there too.

**Comment by fatboypunk** [comment]
> I thought it would be nice to add a Google+ icon there too.

might be, but it's not something we focus on right?

**Comment by RonaldDijkstra** [comment]
True that.

**Comment by fatboypunk** [comment]
> No, it's gonna be in the footer too.

is that going to be included in this PR?

**Comment by RonaldDijkstra** [comment]
Yes

**Comment by RonaldDijkstra** [comment]
I changed the name of my local branch, so I'll just make a new PR.

**Pull request closed without merging**